src/ui/src/services/ipc-client.ts

```diff
@@ -23,6 +23,38 @@ import type {
   YouTubeConfig,
 } from "../types";
 
+export type VideoFileMetadata = {
+  fileName: string;
+  filePath?: string;
+  createdAt: string;
+  duration: number;
+};
+
+export type ShaveStatus = "Pending" | "Processing" | "Completed" | "Failed";
```

Comment:
this seems like it should be a shared type between frontend / backend, correct?
Ideally we have a shared type folder rather then duplicating type definition